### ISSUE TITLE
event.target emitted on click of filter, depends on where you are clicking 

### ISSUE THREAD

**Issue by swathip88** [eBay/ebayui-core#1082]
# Bug Report

## Description
event.target emitted on click of filter, depends on where you are clicking inside the filter. If I click outside the text, I get `button` as `target`. But if I click on the text I get `span` as `target`. ( I am assigning data attribute to button, so need the `target` to be consistent

Consider the example:
```
<ebay-filter
          selected=filter.selected
          w-on-filter-click="quickFilterSelect">
```
and handler looks like this 
```
quickFilterSelect({ originalEvent }) {
    this.emit('select', originalEvent);
 },
```

**Comment by agliga** [comment]
A fix in skin would be to add `.filter-button span { pointer-events: none; }` in order to push events to button. This way the pointer events will bubble up to the parent. 
This would require a skin change as well

**Comment by swathip88** [comment]
Thanks @agliga  for looking into it.. @agliga @ianmcburnie Do you want me to create a feature enhancement ticket on skin?

**Comment by agliga** [comment]
Issue has been released. Also issue has been merged and will be released in `5.2.0`
